.github/workflows/build.yml

```
@@ -240,7 +240,10 @@ jobs:
     strategy:
       matrix:
         # Versions supported by AWS and lambci/lambda images
-        python-version: [3.6, 3.7, 3.8]
+        python-version:
+          - 3.6
+          - 3.7
+          - 3.8
```

Python 3.9 is now supported in AWS Lambda.
```suggestion
        python-version:
          - 3.6
          - 3.7
          - 3.8
          - 3.9
```

---

However there is no image for [lambci/lambda](https://hub.docker.com/r/lambci/lambda/tags?page=1&ordering=last_updated&name=build-python3.9). I would like to keep changes like that to their own PRs as well.

---

Ah, OK.